README.adoc

```diff
@@ -8,15 +8,18 @@ This creates a truly open, highly portable system for developing and maintaining
 
 Complete documentation for LDCMF is still being developed.
 
+This project exhibits a *bootstrapped framework*.
+It is a standardized approach to initiating an AsciiDoc-based docs-as-code project, including an initial file structure and starter files.
+
+=== Quickstart
+
 The following procedure is a *bootstrapping operation* to get an _actual LDCMF project_ started.
 It further sets up initial files based on custom preferences.
 The newly generated `README.adoc` instructs future users to set up and use your docs.
 
-=== Quickstart
-
 [WARNING]
 This file (`README.adoc`) is the README for the generic LiquiDoc CMF project bootstrap repository.
-If you wish to keep this file, move/rename it _before_ running the quickstart, which generates a proper `README.adoc` for your actual docs project/directory, which this directory and its contents then becomes.
+If you wish to keep this file, move/rename it _before_ running the quickstart, which generates a proper `README.adoc` for your actual docs project/directory, which this directory and its contents then _becomes_.
```

Comment:
How about this (edited to say I still don't love it, but this stuck out to me on earlier versions as well so I think we should work on it.)

Following the quickstart generates a final readme to guide your users on interacting with your documentation project/directory. This quickstart process therefore overwrites _this_ Liquidoc CMF-specific README.adoc. If you wish to keep this file, move/rename it _before_ running the quickstart.